openrtb/application_test.go

```diff
@@ -16,70 +14,21 @@ func TestApplicationMarshalUnmarshal(t *testing.T) {
 	openrtbtest.VerifyModelAgainstFile(t, "application.json", ApplicationModelType)
 }
 
-func TestApplication_Copy(t *testing.T) {
-	testCases := []struct {
-		application *openrtb.Application
-	}{
-		{
-			&openrtb.Application{},
-		},
-		{
-			&openrtb.Application{
-				ID:                "testID",
-				Name:              "testName",
-				Bundle:            "testBundle",
-				Domain:            "testDomain",
-				StoreURL:          "testStoreURL",
-				Categories:        []openrtb.Category{openrtb.CategoryArtsAndEntertainment},
-				SectionCategories: []openrtb.Category{openrtb.CategoryArtsAndEntertainment},
-				PageCategories:    []openrtb.Category{openrtb.CategoryArtsAndEntertainment},
-				Version:           "testVersion",
-				HasPrivacyPolicy:  true,
-				IsPaid:            true,
-				Publisher: &openrtb.Publisher{
-					ID:         "testPublisher",
-					Name:       "testName",
-					Categories: []openrtb.Category{openrtb.CategoryAdvertising},
-					Domain:     "testDomain",
-				},
-				Keywords: "testKeywords",
-			},
-		},
+func TestApplication_Fields(t *testing.T) {
+	if err := openrtbtest.VerifyStructFieldNameWithStandardTextFile(
+		(*openrtb.Application)(nil), "testdata/app_std.txt"); err != "" {
```

Comment:
suggest err to be `error` type rather than `string`

Reply:
For VerifyStructFieldNameWithStandardTextFile function, if field names don't match standard text file, that's an error of related struct definition, rather than error of function VerifyStructFieldNameWithStandardTextFile itself. The returned string is normal output of this function.
The confusing point here is the name "err". Maybe we can rename it with "diff" or something like.